src/index.ts

```diff
@@ -1,23 +1,47 @@
 import { type Button, registerPlugin } from '@pexip/plugin-api'
+import { createSelectRoomForm } from './selectRoom'
+import { getMessageOverlay } from './messageOverlay'
+import { createInputMessageForm } from './inputMessage'
+import { setPlugin } from './plugin'
+import { getBreakoutRooms } from './breakoutRooms'
 
 const plugin = await registerPlugin({
   id: 'message-overlay',
-  version: 0
+  version: 1
 })
 
+setPlugin(plugin)
+
 let button: Button<'settingsMenu'> | null = null
 let creatingButton = false
 
+const breakoutRooms = getBreakoutRooms()
+
+plugin.events.authenticatedWithConference.add(async () => {
+  breakoutRooms.clear()
+})
```

Comment:
what happens if we don't have this 😄

Reply:
If you don't do this, leave the room and connect again, you will see the old breakout rooms when you try to send a new message overlay.